README.md

```diff
@@ -456,3 +457,36 @@ after calling `camera.getPicture`. Applies only when the value of
     function onFail(message) {
         alert('Failed because: ' + message);
     }
+
+## navigator.camera.checkForSavedResult
+
+call this method in onDeviceReady to see if there is a result present from a previous call to getPicture.
+add the same options you used when calling getPicture, or leave it out to pass default options
```

Comment:
Capitalize first word. Also, options are required, but you can pass an empty object to get the defaults.